update Sentry Basics for alert wizard changes (#3423)
getsentry/sentry-docs: src/docs/product/sentry-basics/guides/alert-notifications/index.mdx

## Alerts

When you create a new project in [sentry.io](https://sentry.io), you can choose from a few default alerts, or choose to create alerts at a later time. Choosing a default alert, such as getting notified only once when an error is first seen, is useful for catching new errors in real time, but means Sentry will not alert you about subsequent instances of the same error. With only the default alert rule in place, you might be losing visibility into critical issues building up in your code.

> Instead, we recommend you create your own alerts to provide you with the information that your team needs at a useful frequency. To create your own alert, navigate to **Alerts** and click "Create New Alert". Select your project, and then select from the alerting options under "Errors" or "Performance".

Reviewer: ```suggestion
Instead, we recommend creating your own alerts to provide information that your team needs at a frequency that is useful to your team's triage process. To create your own alert, navigate to **Alerts** and click "Create New Alert". Select your project, then select from the alerting options under "Errors" or "Performance".
```

> ![Choice between alerts about Errors, Performance, or a Custom Metric](create-alert-rule)

Author: ```suggestion
![Choice between alerts about Errors, Performance, or a Custom Metric](create-alert-rule.jpg)
```
